NitroxPatcher/Patches/Dynamic/Drillable_Restore_Patch.cs

```diff
@@ -0,0 +1,18 @@
+using System.Reflection;
+using Nitrox.Model.DataStructures;
+using NitroxClient.GameLogic;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public sealed partial class Drillable_Restore_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((Drillable t) => t.Restore());
+
+    public static void Postfix(Drillable __instance)
```

Comment:
Afais This is only used in the AnteChamber. Would maybe add a comment if it is that way. On the other hand, do we need to check for locks here or is this done on the antechamber side?

Reply:
In theory we would need to check for locks here but it doesn't really matter. Even if each player sends this packet it can't start a packet loop or cause a desync, and this case is infrequent enough for the bandwidth use to be negligible. Drillable entities don't use locks at all, so I'm inclined to leave it as is so we don't risk accidental desync from no one having the lock.